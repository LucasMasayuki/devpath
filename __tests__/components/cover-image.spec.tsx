import React from 'react'
import renderer from 'react-test-renderer';
import CoverImage from '@/components/cover-image';

describe('CoverImage', () => {
  it('should render CoverImage', () => {
    const component = renderer.create(
      <CoverImage src='test' title='test'/>,
    )
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})