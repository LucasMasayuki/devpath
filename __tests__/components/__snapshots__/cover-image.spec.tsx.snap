// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CoverImage should render CoverImage 1`] = `
<img
  alt="Cover Image for test"
  className="chakra-image css-2p2hr9"
  loading="lazy"
  src="test"
/>
`;
